tests/unit/amo/pages/TestGuides.js

```diff
@@ -0,0 +1,96 @@
+import * as React from 'react';
+
+import NotFound from 'amo/components/ErrorPage/NotFound';
+import { fetchGuidesAddons } from 'amo/reducers/guides';
+import {
+  dispatchClientMetadata,
+  fakeI18n,
+  shallowUntilTarget,
+} from 'tests/unit/helpers';
+import Guides, { extractId, GuidesBase, getContent } from 'amo/pages/Guides';
+
+describe(__filename, () => {
+  let content;
+  let store;
+  let slug;
+  let i18n;
+  let guids;
```

Comment:
I am not a fan of global variables in test files. This is super tricky to refactor.

Reply:
Can you write a `getProps` function instead? That function can have all the defaults set in `beforeEach`.

Reply:
There is no need to keep the `beforeEach()`